Prediction_Eval.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
```

Comment:
<img width="670" alt="image" src="https://github.com/geo-smart/swe-workflow-book/assets/45752727/f24cf99d-dc61-419a-80e1-e0e85ecf0576">

on the prediction, we need to show the output log of the code.